.github/workflows/test-node.yaml

```diff
@@ -22,3 +22,37 @@ jobs:
         run: |
           npm install prettier --prefix ./pkg/new-ui/v1beta1/frontend
           make prettier-check
+
+  frontend-unit-tests:
+    name: Frontend Unit Tests
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
```

Comment:
Can you rebase this PR and upgrade this `actions/checkout` version to v3 since we upgrade the `actions/checkout` version in #1996 to resolve the warning?

Reply:
Sure, I just force-pushed the branch with both of your comments fixed. Thanks for noticing @tenzen-y

Reply:
I'm ok with upgrading the` actions/setup-node` version. Thanks!